__tests__/unit/core-magistrate-transactions/general.test.ts

```diff
@@ -0,0 +1,87 @@
+import "jest-extended";
+
+import passphrases from "@arkecosystem/core-test-framework/src/internal/passphrases.json";
```

Comment:
Always use `@packages` instead of `@arkecosystem` or jest won't be able to collect coverage.